templates/netlify/remix.init/README.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to Remix!
+
+- [Remix Docs](https://remix.run/docs)
+- [Netlify Edge Functions Overview](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions)
```

Comment:
TODO: @nickytonline update the Edge template with this, not the regular function template.